src/utils/cookieConsentManagement.js

```diff
@@ -0,0 +1,59 @@
+import * as Scrivito from "scrivito";
+
+let resolveCookie;
+let resolveCookieConsentCalledBefore = false;
+
+const cookiePromise = new Promise((resolve) => {
+  resolveCookie = resolve;
+});
+
+function resolveCookieConsent() {
```

Comment:
* [ ] According to https://stackoverflow.com/a/29491617 this is not needed. `resolveCookie()` should resolve only once, regardless of the number of calls.